css-view-transitions-1/Overview.bs

```diff
@@ -363,39 +363,39 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 	This area is consistent regardless of root scrollbars or interactive widgets.
 
 	Issue: "Interactive widgets" refers to UI described within a property definition in [[css-viewport#interactive-widget-section]].
 	This should be extracted into an exported definition that can be used independently of the property.
 
 	<figure>
 		<img src="diagrams/phone-browser.svg" width="200" height="335" alt="A diagram of a phone screen, including a top status bar, a browser URL bar, web-content area with a floating scrollbar, a virtual keyboard, and a bottom bar with an OS back button">
-		<img src="diagrams/phone-browser-snapshot-viewport.svg" width="200" height="335" alt="The previous diagram, but highlights the area that's the 'snapshot' viewport, which includes everything except the top status bar and the bottom bar with the OS back button">
+		<img src="diagrams/phone-browser-snapshot-root.svg" width="200" height="335" alt="The previous diagram, but highlights the area that's the 'snapshot root', which includes everything except the top status bar and the bottom bar with the OS back button">
 		<figcaption>
-			An example of the [=snapshot viewport=] on a mobile OS.
+			An example of the [=snapshot root=] on a mobile OS.
 			The snapshot includes the URL bar, as this can be scrolled away.
 			The keyboard is included as this appears and disappears.
-			The top and bottom bars are part of the OS rather than the browser, so they're not included in the snapshot viewport.
+			The top and bottom bars are part of the OS rather than the browser, so they're not included in the snapshot root.
 		</figcaption>
 	</figure>
 
 	<figure>
 		<img src="diagrams/desktop-browser.svg" width="132" height="79" alt="A diagram of a desktop browser window, including a tab bar, a URL bar, and a web-content area featuring both horizontal and vertical scrollbars" style="height:auto; width: 600px">
-		<img src="diagrams/desktop-browser-snapshot-viewport.svg" width="132" height="79" alt="The previous diagram, but highlights the area that's the 'snapshot' viewport, which includes the web content area and the scrollbars" style="height:auto; width: 600px">
+		<img src="diagrams/desktop-browser-snapshot-root.svg" width="132" height="79" alt="The previous diagram, but highlights the area that's the 'snapshot root', which includes the web content area and the scrollbars" style="height:auto; width: 600px">
 		<figcaption>
-			An example of the [=snapshot viewport=] on a desktop OS.
+			An example of the [=snapshot root=] on a desktop OS.
 			This includes the scrollbars, but does not include the URL bar, as web content never appears in that area.
 		</figcaption>
 	</figure>
 
 	This means the snapshot canvas size is likely to be consistent for the [=document element=]'s [=captured element/old image=] and [=captured element/new element=].
 
-	The <dfn>snapshot viewport origin</dfn> refers to the start of the block and inline axes of the [=snapshot viewport=].
```

Comment:
This is used only in reference to transforms which AFAIK are always based on x,y running positive starting from the top-left.